lib/topological_inventory/sync/host_inventory_sync_worker.rb

```diff
@@ -133,6 +135,15 @@ def perform(message)
         metrics&.record_error(:request)
       end
 
+      def rhel?(host)
+        host["guest_info"] == "Red Hat BYOL Linux" ||
+          host["guest_info"] == "Red Hat Enterprise Linux"
+      end
+
+      def marketplace?(host)
+        host["guest_info"] != "Red Hat BYOL Linux"
```

Comment:
So just validating I'm understanding that I understand this correctly:

A host is RHEL if it matches those 2 strings at L139-L140, and the marketplace string is `"Red Hat Enterprise Linux"`? 

I guess I'm just trying to figure out which one is marketplace - my initial instinct was that BYOL == marketplace. If that's the case can we switch the `!=` to an `==` with the other string? I think it would be easier to understand.

Reply:
BYOL = Bring Your Own License - so we know it's NOT marketplace (licensed through Amazon).
I prefer having the negative logic with this string, just in case they add other strings that also indicate marketplace - BYOL will always indicate not marketplace.

Reply:
Ahhh ok BYOL I thought was Bring Your Own Linux like a custom disk image....License makes a lot more sense here. Now it makes sense!!